### PR TITLE
Upgrade to 1.0.1 and support for systemd

### PR DESCRIPTION
**Pull request by vpiserchia**

### CODE REVIEW

File: app/controllers/Logkafka.scala [hunk 1]
```diff
@@ -137,6 +139,7 @@ class Logkafka (val messagesApi: MessagesApi, val kafkaManagerContext: KafkaMana
           case Kafka_0_11_0_0 => (defaultCreateForm.fill(kafka_0_11_0_0_Default), clusterContext)
           case Kafka_0_11_0_2 => (defaultCreateForm.fill(kafka_0_11_0_2_Default), clusterContext)
           case Kafka_1_0_0 => (defaultCreateForm.fill(kafka_1_0_0_Default), clusterContext)
+          case Kafka_1_0_1 => (defaultCreateForm.fill(kafka_1_0_0_Default), clusterContext)
```
Comment (sm4rk0) [inline app/controllers/Logkafka.scala]: shouldn't this be kafka_1_0_1_Default?
```
case Kafka_1_0_1 => (defaultCreateForm.fill(kafka_1_0_1_Default), clusterContext)
```

**Comment by sm4rk0** [comment]
Looking at the source, [here](https://github.com/yahoo/kafka-manager/tree/master/app/kafka/manager/utils) are zero81, zero82, zero90 packages with related files copied from the respective Kafka version (zero81 -> 0.8.1, zero82 -> 0.8.2, zero90 -> 0.9.0), but newly added "supported" versions from 0.10 and up only refer to zero90. There are no new packages from newer Kafka versions. Is this really OK? Or it just works because newer Kafka versions are backward compatible.
Here's the original LogConfig.scala, if someone wants to analyze:
https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/log/LogConfig.scala

File: app/controllers/Logkafka.scala (hunk 1 above)
Comment (lord2y) [reply app/controllers/Logkafka.scala]: yes, you're right.

**Comment by vpiserchia** [comment]
I'll will resubmit a new patch series, with the a proper uno01/LogConfig.scala attached to it; possibly also a uno00.
ty

**Comment by lord2y** [comment]
Yes. @vpiserchia and will work on it to add what’s missing

**Comment by jthompson6** [comment]
Thanks.

**Pull request closed without merging**